packages/ecc-client-ga4gh-trs/src/components/trs-list/custom/copy/copy.template.ts

```diff
@@ -0,0 +1,31 @@
+import { html } from "@microsoft/fast-element";
+import {
+  fastButton,
+  provideFASTDesignSystem,
+} from "@microsoft/fast-components";
+import type { CopyClipboard } from "./copy.js";
+import { copyIcon, doneIcon } from "../../../../assets/icons.js";
+
+provideFASTDesignSystem().register(fastButton());
+
+export const template = html<CopyClipboard>`
+  <head>
+    <script
+      type="module"
+      src="https://cdn.jsdelivr.net/npm/@microsoft/fast-components/dist/fast-components.min.js"
+    ></script>
+  </head>
```

Comment:
Not the correct way to load the components in your components. Do it something like this: https://github.com/elixir-cloud-aai/cloud-components/blob/main/packages/ecc-client-ga4gh-tes/src/components/tes-run/definition/tesRun.template.ts